Formula/ssh-vault.rb

```diff
@@ -0,0 +1,32 @@
+class SshVault < Formula
+  desc "encrypt/decrypt using ssh keys"
+  homepage "https://ssh-vault.com/"
+  url "https://github.com/ssh-vault/ssh-vault.git",
+      :tag => "0.12.2",
+      :revision => "39c919577ab522293750d1c01174a8b5b9871aae"
+
+  head "https://github.com/ssh-vault/ssh-vault.git"
+
+  depends_on "dep" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/ssh-vault/ssh-vault").install buildpath.children
+    cd "src/github.com/ssh-vault/ssh-vault" do
+      system "dep", "ensure"
+      ldflags = "-s -w -X main.version=#{version}"
+      system "go", "build", "-ldflags", ldflags, "-o", "#{bin}/ssh-vault", "cmd/ssh-vault/main.go"
+      prefix.install_metafiles
+    end
+  end
+
+  test do
+    system bin/"ssh-vault", "-v"
```

Comment:
You can drop the `system bin/"ssh-vault", "-v"` now that you have the other tests.

Reply:
done, many thanks 👍